tools/odbc/test/tests/select.cpp

```diff
@@ -52,6 +50,16 @@ TEST_CASE("Test Select Statement", "[odbc]") {
 		DATA_CHECK(hstmt, i, std::to_string(i).c_str());
 	}
 
+	// SELECT $x; should throw error
+	SQLRETURN ret = SQLExecDirect(hstmt, ConvertToSQLCHAR("SELECT $x"), SQL_NTS);
+	REQUIRE(ret == SQL_ERROR);
+	std::string state;
+	std::string message;
+	ACCESS_DIAGNOSTIC(state, message, hstmt, SQL_HANDLE_STMT);
+	REQUIRE(state == "42000");
+	REQUIRE(message == "ODBC_DuckDB->PrepareStmt\n"
```

Comment:
Not specifically related to this PR - but in the past I have found it potentially problematic to have exact error message comparisons because minor changes in error messages (e.g. reformatting `ODBC_DuckDB->PrepareStmt`) would require all tests to be rewritten.

Perhaps change this to something along the lines of `REQUIRE(StringUtil::Contains(message, "Not all parameters are bound"))`?

Reply:
Good point, I've actually done this in a couple of other tests so I think I'll do a separate PR to change them all